packages/notify-client/src/controllers/engine.ts

```diff
@@ -1009,7 +1028,9 @@ export class NotifyEngine extends INotifyEngine {
     });
   };
 
-  private resolveDappPublicKey = async (dappUrl: string): Promise<string> => {
+  private resolveDappPublicKey = async (
```

Comment:
The method name is misleading now if we return two distinct keys, we should rename in this case:

```suggestion
  private resolveDappKeys = async (
```